specification/protocol/open_inference_rest.yaml

```
@@ -135,7 +179,44 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/inference_request'
-      description: 'An inference request is made with an HTTP POST to an inference endpoint. In the request the HTTP body contains the [Inference Request JSON Object](#inference-request-json-object). In the corresponding response the HTTP body contains the [Inference Response JSON Object](#inference-response-json-object) or [Inference Response JSON Error Object](#inference-response-json-error-object). See [Inference Request Examples](#inference-request-examples) for some example HTTP/REST requests and responses.'
+      description: >
+        An inference request is made with an HTTP POST to an inference endpoint.
+        Compliant servers return an [Inference Response JSON Object](#inference-response-json-object) or an [Inference Response JSON Error Object](#inference-response-json-error-object).
+        See [Inference Request Examples](#inference-request-examples) for some example HTTP/REST requests and responses.
+        The model name and version must be provided in the URL.
```

Rewriting this section to still include the link to the input format, but talk less about the details of HTTP/REST.

```suggestion
        Send data to a model for inferencing via an [Inference Request JSON Object](#inference-request-json-object).
        Compliant servers return an [Inference Response JSON Object](#inference-response-json-object) or an [Inference Response JSON Error Object](#inference-response-json-error-object).
        The model name and version must be provided in the URL.

        See [Inference Request Examples](#inference-request-examples) for some example HTTP/REST requests and responses.
```